spec/features/email_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'User can send emails' do
+  it 'From obituary show page' do  
+
+    user = create(:user)
+    obit = create(:obituary)
+
+    visit '/'
+    click_on "Login"
+
+    fill_in :email, with: user.email
+    fill_in :password, with: user.password
+    find('#submit').click
+
+    visit "obituaries/#{obit.id}"
+
+    click_on 'Share this Obituary in an Email'
+    expect(current_path).to eq("/obituaries/#{obit.id}/share")
+
+    fill_in :recipient_email, with: "bob@bob.com"
+    fill_in :recipient_name, with: "bob"
+    click_on "Send Email"
+
+    email = ActionMailer::Base.deliveries.last
+    expect(email.from).to have_content(["obituaryhub@gmail.com"])
```

Comment:
For some reason these are coming back as an array, couldnt figure out why, doesnt seem to be causing any issues.